source/cloud-security/amazon/services/prerequisites/considerations.rst

```diff
@@ -270,23 +270,13 @@ Enabling dashboard visualization
 You can activate the corresponding Security Information Management module on the Wazuh Dashboard. This module provides additional details and insights about events, as shown in the screenshots below.
```

Comment:
**nitpick**: review the `Enabling dashboard visualization` if it is required or not.

Reply:
I will remove the reference to enabling the module and the related images.

Reply:
Fixed in ef02e7cd655d1412e5f49256400d4ba4196357cc.